app/domain/authentication/resource_restrictions/extract_resource_restrictions.rb

```diff
@@ -31,29 +34,16 @@ def call
 
       private
 
-      def fetch_resource_annotations
+      def call_fetch_resource_annotations
```

Comment:
why was this name changed? i think that `fetch_resource_annotations` is better

Reply:
Done